### PR TITLE
Implement module to generate the most frequent words from a text

### PR DESCRIPTION
**Pull request by divyanshukatiyar**
Implements #240

### CODE REVIEW

File: generators/python_functions/most_frequent_words/code_snippet.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+```python
+from collections import Counter
+
+YOUR_ATTRIBUTE: str = "text"  # only text fields
+YOUR_N_WORDS: int = 5
+
+def most_frequent_words(record):
+    
+    words = [token.text for token in record[YOUR_ATTRIBUTE] if not token.is_stop and not token.is_punct]
+    return Counter(words).most_common(YOUR_N_WORDS)
```
Comment (LeonardPuettmann) [inline generators/python_functions/most_frequent_words/code_snippet.md:10]: This returns a list of tuples, which we cannot work within refinery at the moment. Doing it like this worked for me: `return str(Counter(words).most_common(YOUR_N_WORDS)).strip("[]")`

File: generators/python_functions/most_frequent_words/__init__.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+def most_frequent_words(request: MostFrequentWordsModel):
+    """Generates the frequency of the words and shows top n words"""
+
+    text = request.text
+    nlp = SpacySingleton.get_nlp(request.spacyTokenizer)
+    doc = nlp(text)
+
+    words = [token.text for token in doc if not token.is_stop and not token.is_punct]
+
+    return {"frequent": Counter(words).most_common(request.n_words)}
```
Comment (LeonardPuettmann) [inline generators/python_functions/most_frequent_words/__init__.py:30]: Nit-picking, but maybe write "frequent_words" or something here? :)

File: generators/python_functions/most_frequent_words/__init__.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+from pydantic import BaseModel
+from generators.util.spacy import SpacySingleton
+from collections import Counter
+
+INPUT_EXAMPLE = {
+    "text": "APPL went down by 5% in the past two weeks. Shareholders are concerned over the continued recession since APPL and NASDAQ have been hit hard by this recession. Risks pertaining to short-selling are pouring in as APPL continues to depreciate. If the competitors come together and start short-selling, the stock can face calamity.",
```
Comment (LeonardPuettmann) [inline generators/python_functions/most_frequent_words/__init__.py:6]: Really like the idea of the module. Extract the most frequent words and then for a search with Google/ Bing to enrich your data quickly. Really simple but powerful! Will probably use this for the next video. :)

File: generators/python_functions/most_frequent_words/__init__.py (hunk 2 above)
Comment (divyanshukatiyar) [reply generators/python_functions/most_frequent_words/__init__.py:30]: Yeah I just did that for dummy, obviously would count on more than my opinions :)

File: generators/python_functions/most_frequent_words/code_snippet.md (hunk 1 above)
Comment (divyanshukatiyar) [reply generators/python_functions/most_frequent_words/code_snippet.md:10]: Yes, I'll test that thoroughly and fix it :)

File: generators/python_functions/most_frequent_words/__init__.py (hunk 3 above)
Comment (divyanshukatiyar) [reply generators/python_functions/most_frequent_words/__init__.py:6]: Running on refinery, it looks so very powerful!!!

**Review by LeonardPuettmann — APPROVED** [review]

**Pull request merged**